logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -448,8 +468,16 @@ class ProtoContentMapperImpl(
                 unpackAsset(genericAssetContent)
             }
 
+            is Ephemeral.Content.Knock -> {
+                MessageContent.Knock(
+                    ephemeralContent.value.hotKnock
+                )
+            }
+
             // Handle self-deleting Location messages when they are implemented
-            else -> {
+            is Ephemeral.Content.Image,
```

Comment:
Should `Image` be ignored as well? 🤔

Reply:
not sure what image is in this case, it was ignored before and my assumption that it is some sort of old and deprecated message type, maybe @typfel knows more about it

Reply:
Indeed it was replaced by assets  https://github.com/wireapp/generic-message-proto/blob/ff36abddb9d668709e0c793ddc8d034ebfafd8f9/proto/messages.proto#L29 7 years ago.